c_plus_plus/src/model/stream/audio/audio_config.h

```diff
@@ -0,0 +1,33 @@
+#ifndef AUDIO_CONFIG_H
+#define AUDIO_CONFIG_H
+
+#include <string>
+
+#include "model/stream/utils/images/image_format.h"
+#include "model/stream/utils/models/dim2.h"
+
+namespace Model
+{
+struct AudioConfig
+{
+    AudioConfig(std::string deviceName, int channels, int rate, int formatBytes, bool isLittleEndian, int bufferSize)
```

Comment:
Use const reference when passing strings and the std::move is not necessary here (your doing a copy + a move instead of a single copy)